src/plugins/intel_cpu/src/nodes/mkldnn_softmax_node.cpp

```diff
@@ -10,11 +10,44 @@
 #include <memory_desc/cpu_memory_desc_utils.h>
 #include <ngraph/opsets/opset1.hpp>
 #include "memory_desc/dnnl_blocked_memory_desc.h"
+#include <common/primitive_hashing_utils.hpp>
 
 using namespace mkldnn;
 using namespace MKLDNNPlugin;
 using namespace InferenceEngine;
 
+namespace {
+struct SoftmaxKey {
+    DnnlMemoryDescCPtr inp0;
+    impl_desc_type implType;
+    size_t axis;
+
+    size_t hash() const;
+    bool operator==(const SoftmaxKey& rhs) const;
+};
+
+size_t SoftmaxKey::hash() const {
+    using namespace dnnl::impl;
+    using namespace dnnl::impl::primitive_hashing;
+
+    size_t seed = 0;
+
+    seed = hash_combine(seed, get_md_hash(inp0->getDnnlDesc().data));
+    seed = hash_combine(seed, implType);
```

Comment:
Since `axis` is a part of the key, we have to add it to the hash calculation.

Reply:
yep, it should be added